GPflow/param.py

```diff
@@ -305,8 +305,7 @@ def randomize(self, distributions={}, skipfixed=True):
                 try:
                     self._array = self.prior.sample(self.shape)
                 except AttributeError:
-                    randn = np.random.randn(
-                        self.transform.free_state_size(self.shape))
+                    randn = np.random.normal(size=self.shape)
```

Comment:
@mrksr , this won't work for transforms that don't preserve size, I think @Bonnevie is right here.

The problem is below, where we need:

`self._array = self.transform.forward(randn).reshape(self.shape)`